site/src/Map.jsx

```diff
@@ -329,6 +328,15 @@ export default function Map({ mode, mapEntity, setMapEntity, setZoom }) {
 
           <ThemeSelector visibleThemes={setVisibleThemes}></ThemeSelector>
         </div>
+        <div className="bug-nub">
+          <a
+            className="bug-nub-link"
+            href="https://github.com/OvertureMaps/io-site/issues/new/choose"
```

Comment:
Note that this `href` may need to be tested after the templates are added to this git repo. Since we have no templates currently, it is difficult to be 100% sure what the true `href` will be.

Reply:
This is great! Feel free to create an issue to remind yourself to do this work, then attack that issue once the templates land.